doc/storage.md

```diff
@@ -13,9 +13,52 @@ are block devices that:
 * do not contain partition information
 * have more than 1 GiB
 
-Any such device will be prepared according to the value of `operator.satelliteSet.automaticStorageType`. Devices are
-added to a storage pool based on the device name (i.e. all `/dev/nvme1` devices will be part of the pool
-`nvme1`)
+To enable automatic configuration of devices, set the `devicePaths` key on `storagePools` entries:
+
+```yaml
+  storagePools:
+    lvmPools:
+    - name: lvm-thick
+      volumeGroup: drbdpool
+      devicePaths:
+      - /dev/vdb
+    lvmThinPools:
+    - name: lvm-thin
+      thinVolume: thinpool
+      volumeGroup: linstor_thinpool
+      devicePaths:
+      - /dev/vdc
+      - /dev/vdd
+```
+
+Currently, this method support creation of LVM and LVMTHIN storage pools.
```

Comment:
Perhaps we should add ZFS support at the same time. Otherwise we are deprecating the method to create ZFS pools without replacing it.

Reply:
I actually started implementing it, but I'm pretty sure it never worked in the first place. There is some interaction between the ZFS kmod and the tools that does not seem to work in containers when I tried.

Reply:
Fair enough. If it never worked, no harm in deprecating it for now.